### ISSUE TITLE
`x` close button using the wrong colour while using the dark theme

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#21574]
## Description 

The `X` close button under the `brave://wallet` promo card which was introduced via https://github.com/brave/brave-browser/issues/21224 is appearing as `white` while using the `dark` theme.

## Steps to Reproduce

Prerequisite: ensure that your locale is set as `US` as the promo card is US specific

   1. launch `1.36.111 Chromium 99.0.4844.51`
   2. enable `brave://wallet` (should notice the promo card)
   3. change the theme to `dark` via `brave://settings/appearance`

## Actual result:

![image](https://user-images.githubusercontent.com/2602313/157529822-c9703ee6-48e1-489b-a393-d059f5df9cc4.png)

## Expected result:

![image](https://user-images.githubusercontent.com/2602313/157529848-2e6247ab-fd39-49f2-b6c3-eeccaaa41311.png)

## Reproduces how often: 

100% reproducible using the STR/Cases outlined above.

## Brave version (brave://version info)

```
1.36.111 Chromium 99.0.4844.51
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? `Yes`
- Can you reproduce this issue with the beta channel? `Yes`
- Can you reproduce this issue with the nightly channel? `Yes`

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? `N/A`
- Does the issue resolve itself when disabling Brave Rewards? `N/A`
- Is the issue reproducible on the latest version of Chrome? `N/A`

**Comment by kjozwiak** [comment]
Closing as this was fixed via https://github.com/brave/brave-core/pull/12504. Created the above issue as the PR was originally pointing to https://github.com/brave/brave-browser/issues/21224 which was already released. The above PR/uplifts were created after the fact so a new issue should have been created.

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x64` using the following build(s):

```
Brave | 1.36.112 Chromium: 99.0.4844.51 (Official Build) (64-bit)
-- | --
Revision | d537ec02474b5afe23684e7963d538896c63ac77-refs/branch-heads/4844@{#875}
OS | Windows 10 Version 21H2 (Build 19044.1566)
```

**`Example (Light Theme)`** | **`Example (Dark Theme)`**
--------------------------------|-------------------------------
![image](https://user-images.githubusercontent.com/2602313/157598825-35988353-a835-4cfb-acf5-b15e6640eeb9.png) | ![image](https://user-images.githubusercontent.com/2602313/157598874-d0d01cb3-763e-42b2-8816-8d79a8e43189.png)
